src/app/content/components/Page/PageComponent.tsx

```diff
@@ -37,6 +38,20 @@ export default class PageComponent extends Component<PagePropTypes> {
   private processing: Array<Promise<void>> = [];
   private componentDidUpdateCounter = 0;
 
+  private linksToOtherPagesOpenInNewTab(rootEl: HTMLElement) {
```

Comment:
ok you moved it to a function, but i would really like to get it out of this file, this file has a lot of complexity already. can you please move to the [contentDOMTransformations.ts](https://github.com/openstax/rex-web/blob/ecb62adf6d2bad690953b5b31a3665d5f5058e23/src/app/content/components/Page/contentDOMTransformations.ts) file? you will still need to call it directly because of the extra parameters (unless you feel like adding additional props to the `transformContent` function), but even just moving the function would be nice